### PR TITLE
build(chocolatey): release with version

### PR DESCRIPTION
**Pull request by morrieinmaas**
Adds automatic chocolatey release to tags builds

### CODE REVIEW

**Comment by morrieinmaas** [comment]
@janrtvld when this is merged can you add choco to the docs?

File: choco/agent-cli.nuspec [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?><package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
```
Comment (blu3beri) [inline choco/agent-cli.nuspec:1]: Can we move the `choco` folder under `pkg`? I'd prefer not too have so much in root.

File: choco/agent-cli.nuspec [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <metadata>
+    <id>agent-cli</id>
+    <version>0.1.0</version>
```
Comment (blu3beri) [inline choco/agent-cli.nuspec:4]: If we dynamically replace this version, can we use 0.0.0?

**Review by blu3beri — CHANGES_REQUESTED** [review]

**Comment by morrieinmaas** [comment]
@blu3beri thanks for the feedback. feedback is addressed.

**Review by blu3beri — APPROVED** [review]

**Pull request merged**